web/templates/user_group_settings/user_group_settings_overlay.hbs

```diff
@@ -22,7 +22,7 @@
                 <div class="input-append group_name_search_section" id="group_filter">
                     <input type="text" name="group_name" id="search_group_name" class="filter_text_input" autocomplete="off"
                       placeholder="{{t 'Filter groups' }}" value=""/>
-                    <button type="button" class="btn clear_search_button" id="clear_search_group_name">
+                    <button type="button" class="bootstrap-btn clear_search_button" id="clear_search_group_name">
```

Comment:
Actually seeing this commit, renaming `bootstrap-btn`, and how many of the uses have some other class on the button, like `clear_search_button`, I think we can use this commit as a short TODO list for being able to delete `bootstrap-btn` as well. Half of them are `clear_search_button`, the `bot_avatar_row.hbs` file is scheduled to be deleted in https://github.com/zulip/zulip/pull/31164, and it seems like there's at most a handful of other elements using these.

So maybe you can do a follow-up PR that cleans up the `clear_search_button` styles in addition to the recent table thing noted above, and we can wait for #31164 to merge and then it'll be easy to finish deleting that chunk of Bootstrap!